dyno/panda/dyno_panel.py

```diff
@@ -63,20 +87,50 @@ def update_labels():
 
 #Define on_click function for button
 def handle_click(event):
-    if button["text"] == "Pause":
-        button["text"] = "Run"
+    global FLAG_RUN, text_file_data
+
+    if start_button["text"] == "Pause":
+        start_button["text"] = "Run"
         if READ_SERIAL_ON:
             driver.write('P;')
         print("Flag was set to 'P;'")
+        FLAG_RUN = False
         writeFile()
         update_labels()
-    elif button["text"] == "Run":
-        button["text"] = "Pause"
+    elif start_button["text"] == "Run":
+        start_button["text"] = "Pause"
+        text_file_data = []
         if READ_SERIAL_ON:
             driver.write('R;')
+        FLAG_RUN = True
         print("Flag was set to 'R;'")
 #---------------------
 
+def push_config(event):
+    magnet_config = "magnets="
+    for i in range(num_magnets):
+        temp = magnet_tkvars[i].get()
+        if int(temp) < 0:
+            temp = "0"
+        elif int(temp) > 9:
+            temp = "9"
+        magnet_config = magnet_config + str(i) + ":00" + temp + ","
```

Comment:
Is temp the new setting for the magnet? i.e., is temp the value we want to be mapping to the PWM value that we drive that particular magnet with? If so, maybe pwm is a better name for it?

Also, the Arduino implementation currently expects a PWM value in [0, 255], not [0, 10]. I think we should be mapping that value to [0, 255] here before pushing it to the Arduino because that gives us the flexibility to change the granularity with which the user can adjust the strength of the magnet using this panel going forward, without having to change the Arduino's interface. Does that make sense?